landing-page/scripts/tasks_encode.py

```
@@ -79,30 +81,32 @@ def _resolve_hints(project_id: Optional[int], image_name: Optional[str], page_w,
                 cur.execute(
-                    "SELECT id, yolo_txt FROM annotations WHERE image_name = %s AND project_id = %s "
+                    "SELECT id, yolo_txt, image_id FROM annotations WHERE image_name = %s AND project_id = %s "
                     "ORDER BY created_at DESC LIMIT 1",
                     (image_name, project_id),
                 )
                 row = cur.fetchone()
                 if row:
-                    current_annotation_id, current_yolo_txt = row
+                    current_annotation_id, current_yolo_txt, current_image_id = row
             except Exception:
                 pass
```

Ruff (0.16.1)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)